serverless.yml

```
@@ -2,21 +2,27 @@ service: sound-id-finder-bot
 
 provider:
   name: aws
-  runtime: python3.6
+  runtime: python3.8
   stage: dev
```

добавь тут еще вот такое 

```suggestion
  stage: dev
  memorySize: 128
  timeout: 30
```